bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/AngularVelocity.scala

```diff
@@ -22,10 +22,17 @@ case class DeclinationAngularVelocity(velocity: AngularVelocity)
 
 object AngularVelocity {
 
+  /**
+    * Builds an AngularVelocity expressed on degrees per year
+    */
+  def fromDegreesPerYear(dpy: Double): AngularVelocity =
+    AngularVelocity(dpy * DegreesYearToArcSecsYear)
```

Comment:
Doesn't the `AngularVelocity` constructor expect a `Double` representing mas, not arcsec?